core/src/main/resources/hudson/model/AbstractBuild/index.jelly

```diff
@@ -48,7 +48,7 @@ THE SOFTWARE.
         </div>
       </div>
 
-      <t:buildCaption>${%Build} ${it.displayName} (<i:formatDate value="${it.timestamp.time}" type="both" dateStyle="medium" timeStyle="medium"/>)</t:buildCaption>
+      <t:buildCaption>${it.displayName} (<i:formatDate value="${it.timestamp.time}" type="both" dateStyle="medium" timeStyle="medium"/>)</t:buildCaption>
```

Comment:
On the build page the icon is now selectable but does nothing when clicked. Wouldn't it make sense to either remove the clickable (and selectable) state or navigate to the console log (as in the build history)?